newrelic-agent/src/main/java/com/newrelic/agent/instrumentation/ClassTransformerServiceImpl.java

```diff
@@ -157,6 +159,10 @@ private PointCutClassTransformer startClassTransformer(InstrumentationProxy inst
         // java-io-stream.
         NewRelic.getAgent().getTransaction();
 
+        // Preload Security used classes to avoid complete application thread blocking in rare scenarios.
+        StringUtils.startsWithAny("abc", "z", "b", "a");
```

Comment:
would just setting the empty string here work?

Reply:
yes it will work, if you want i can change

Reply:
Yes, please. I think it will look less hacky.